<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLead</class>
 <widget class="QWidget" name="CreateLead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主角创建</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>人物基本信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QWidget" name="nameAddMajor" native="true">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>290</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="name" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="nameLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>姓名:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameInput">
               <property name="minimumSize">
                <size>
                 <width>135</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string notr="true">别人对你的称呼.</string>
               </property>
               <property name="placeholderText">
                <string>请输入姓名</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="nickName" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="nickNameLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>昵称:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nickNameInput">
               <property name="minimumSize">
                <size>
                 <width>135</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>亲近的人对你的称呼.</string>
               </property>
               <property name="placeholderText">
                <string>请输入昵称</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="claimed" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="ClaimedLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>自称:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="claimedInput">
               <property name="minimumSize">
                <size>
                 <width>135</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>顾名思义,自己对自己的称呼,比如我.</string>
               </property>
               <property name="placeholderText">
                <string>请输入自称</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Occupation" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="OccupationLable">
               <property name="maximumSize">
                <size>
                 <width>29</width>
                 <height>22</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>职业:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="selectOccupation">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>这是你现在的职业,现在并不影响什么.</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Major" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="MajorLable">
               <property name="maximumSize">
                <size>
                 <width>29</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>专业:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="selectMajor">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>21</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>这是你本科的专业,暂时只影响你的技能.</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="birthday" native="true">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>36</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="birthdayLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>生日:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="monthInput">
               <property name="minimumSize">
                <size>
                 <width>72</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string notr="true"/>
               </property>
               <property name="placeholderText">
                <string>请输入月份</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dayInput">
               <property name="minimumSize">
                <size>
                 <width>72</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string notr="true"/>
               </property>
               <property name="placeholderText">
                <string>请输入日期</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="AttributeTemp" native="true">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>290</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="PointsRemaining">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>剩余点数:200</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Willpower" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="WillpowerLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>意志:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="WillpowerInput">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;标准值100,作用对抗催眠、洗脑、交涉、讨价还价等等的行为.可以通过成功抵抗上述行为而增加.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Quick" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="QuickLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>敏捷:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="QuickInput">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>标准值100,作用修正体力消耗和技术.多做做就提升了.做什么,兄弟你要知道这是一个黄油,你说做什么?</string>
               </property>
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Physique" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="PhysiqueLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>体质:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="PhysiqueInput">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>标准值100,影响行动,每次行动都要消耗这个值.可以通过锻炼来增加这个值.</string>
               </property>
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Charm" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="CharmLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>魅力:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CharmInput">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>标准值为100,影响好感度和交涉等.主角可以通过系统提升.</string>
               </property>
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="Intelligence" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="IntelligenceLable">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>智力:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="IntelligenceInput">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="toolTip">
                <string>标准值为100,影响技能的学习.可以通过学习新的技能,或者提升原有技能的等级来提升.</string>
               </property>
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他属性</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
